lib/src/entrypoint.dart

```diff
@@ -170,12 +170,12 @@ class Entrypoint {
 
   /// Loads the entrypoint for the package at the current directory.
   Entrypoint.current(this.cache)
-      : root = Package.load(null, '.', cache.sources, isRootPackage: true),
+      : root = Package.load(null, '.', cache.sources),
```

Comment:
I love that we get rid of `isRootPackage`!!!

Reply:
Yes, it's fantastic :D